### ISSUE TITLE
Unable to set icon (Android)

### ISSUE THREAD

**Issue by alextheedwards** [Paraboly/react-native-card#21]
I'm trying to set the icon at the left of the card

I've tried a few different icons and libraries but can only get either a question mark or a square with a cross in it

Even copying the example code from the docs is giving me this result. 

I've copied my package.json below in case I have missed any imports

```
 {
   "name": "CardTesting",
   "version": "0.0.1",
   "private": true,
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint ."
   },
   "dependencies": {
     "@freakycoder/react-native-bounceable": ">= 0.2.4",
     "@paraboly/react-native-card": "^1.0.3",
     "react": "18.1.0",
     "react-entypo-icons": "^1.4.1",
     "react-native": "0.70.6",
     "react-native-androw": "^0.0.34",
     "react-native-dynamic-vector-icons": ">= 1.1.4",
     "react-native-ionicons": "^4.6.5",
     "react-native-vector-icons": ">= 7.1.0"
   },
   "devDependencies": {
     "@babel/core": "^7.12.9",
     "@babel/runtime": "^7.12.5",
     "@react-native-community/eslint-config": "^2.0.0",
     "babel-jest": "^26.6.3",
     "eslint": "^7.32.0",
     "jest": "^26.6.3",
     "metro-react-native-babel-preset": "0.72.3"
   },
   "jest": {
     "preset": "react-native"
   }
 }
```

When I run the app, i get two logs returned in metro

```
 LOG  Running "CardTesting" with {"rootTag":1}
 LOG  s
```

**Comment by kuraydev** [comment]
Helloo @alexedwards1 
Can you show me how it looks on screen? You should be clear that is it on iOS or Android? Also you should be check the `https://github.com/oblador/react-native-vector-icons#installation` doc for icons

**Comment by alextheedwards** [comment]
I hadn't realised that I needed to do anything to get react-native-vector-icons working...so that has fixed it. Thanks so much for the speedy response.

For anyone else who has the same problem I just needed to add a line of code to the end of my android/app/build.gradle
(The docs say to be careful not to do android/build.gradle by accident)

```
apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"

```

Screenshot to see what it was doing
![image-2.png](https://user-images.githubusercontent.com/73524323/212102863-e5267c7e-76c1-48f8-a6be-0e18a1bc2555.png)

All working now. Thanks again!
